docs/kb/frontend.md

```diff
@@ -1,13 +1,13 @@
 ---
-description: Details on the Vue frontend app created by Starport.
+description: Details on the Vue frontend app created by Ignite CLI.
 order: 8
 ---
 
 # Frontend Overview
 
 A Vue frontend app is created in the `vue` directory when a blockchain is scaffolded. To start the frontend app run `npm i && npm run serve` in the `vue` directory.
 
-The frontend app is built using the `@starport/vue` and `@starport/vuex` packages. For details, see the [monorepo for Starport front-end development](https://github.com/tendermint/vue).
+The frontend app is built using the `@ignite/vue` and `@ignite/vuex` packages. For details, see the [monorepo for Ignite CLI front-end development](https://github.com/tendermint/vue).
```

Comment:
@ilgooz  what is our monorepo now? If in a repo other than ignite-hq/cli, maybe I don't have permission to verify?

Reply:
Let's keep these lines as it was, these packages are not rebranded yet.